service-print/src/main/java/org/oskari/print/request/PDPrintStyle.java

```diff
@@ -92,6 +94,15 @@ public void setIcon(PDFormXObject icon) {
     public void setLabelProperty(List<String> labelProperty) {
         this.labelProperty = labelProperty;
     }
+    public LabelAlign getLabelAlign () {
+        return labelAlign;
+    }
+    public void setLabelAlign (LabelAlign labelAlign) {
+        this.labelAlign = labelAlign;
+    }
+    public void setLabelAlign (String align, float x, float y) {
+        this.labelAlign = new LabelAlign(align, x, y);
```

Comment:
Since LabelAlign only has one public constructor that takes these exact same parameters in the same order this utility setter method adds very little value. Also this is the only utility setter method in the class so it's a bit out of place as the rest of the class is basically a dumb data holder.